apps/web/app/environments/[environmentId]/(actionsAndAttributes)/actions/ActionSettingsTab.tsx

```diff
@@ -23,13 +23,13 @@ import { Controller, useForm } from "react-hook-form";
 import { toast } from "react-hot-toast";
 import { testURLmatch } from "./testURLmatch";
 
-interface EventSettingsTabProps {
+interface ActionSettingsTabProps {
   environmentId: string;
   eventClassId: string;
```

Comment:
`eventClassId` -> `actionClassId`. I think it's best if you review the complete file and replace all occurences of `eventClass` with `actionClass` 😊

Reply:
done, also made an optimisation by passing the entire actionClass from the previous component since we had it there, this saved an extra backend and subsequent DB call